### PR TITLE
Do not merge: Feature/combine partner place organisations

### PR DESCRIPTION
**Pull request by crantok**
* Place and Partner have been merged into Partner.
* Place may well be a better name than Partner but I started with Partner and didn't want to add name-change edits on top of everything else.
* Instead of having places, Partners now have relationships to other Partners. See class OrganisationRelationship
* There is no UI yet for OrganisationRelationship but obvs they can easily be created in the console.
* Contrary to previous discussion, there should be no need to reimport events. Remapping of calendar.place and event.place to the new partners should happen in the database migration.
* No columns or data are destroyed in the migrations. Everything should still be available in one way or another from the DB.

This **really** needs eyes on it to see whether what I've done is sane.

I do **not** advise merging this as a series of commits. I went back and messed with migrations more than once. I advise squashing the whole thing into a single commit before merging.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -42,15 +41,16 @@
   get '/events/:year/:month/:day' => 'events#index', constraints: ymd
 
   # Places
-  resources :places, only: %i[index show]
-  get '/places/:id/events' => 'places#show'
-  get '/places/:id/events/:year/:month/:day' => 'places#show', constraints: ymd
-  get '/places/:id/embed' => 'places#embed'
+  # resources :places, only: %i[index show]
+  # get '/places/:id/events' => 'places#show'
```
[the remark this answers is not in the document]
Comment (crantok) [reply config/routes.rb]: Cool. I added those routes back in, including the embed one.

File: db/migrate/20181009162402_add_organisation_relationships_table.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+class AddOrganisationRelationshipsTable < ActiveRecord::Migration[5.1]
+
+  def change
+    create_table :organisation_relationships
+    add_reference :organisation_relationships, :subject, null: false
+    add_foreign_key :organisation_relationships, :partners, column: :subject_id
+    add_column :organisation_relationships, :verb, :string, null:false
```
[the remark this answers is not in the document]
Comment (crantok) [reply db/migrate/20181009162402_add_organisation_relationships_table.rb]: Which word?

[the remark this answers is not in the document]
Comment (kimadactyl) [reply db/migrate/20181009162402_add_organisation_relationships_table.rb]: ` null:false` needs a space, sorry thought github would highlight my text selection :)

[the remark this answers is not in the document]
Comment (crantok) [reply db/migrate/20181009162402_add_organisation_relationships_table.rb]: OMG! Ruby seems to be very messed up on whether my original version is valid syntax. Compare `003` and `007`. They are the same statement but with different results!

```
$ irb
2.4.3 :001 > {a:false}
 => {:a=>false} 
2.4.3 :002 > puts a:false
{:a=>false}
 => nil 
2.4.3 :003 > puts {a:false}

 => nil 
2.4.3 :004 > a = 1
 => 1 
2.4.3 :005 > puts {a:a}
SyntaxError: (irb):5: syntax error, unexpected ':', expecting '}'
puts {a:a}
        ^
	from /usr/share/rvm/rubies/ruby-2.4.3/bin/irb:11:in `<main>'
2.4.3 :006 > puts a:a
{:a=>1}
 => nil 
2.4.3 :007 > puts {a:false}
SyntaxError: (irb):7: syntax error, unexpected ':', expecting '}'
puts {a:false}
        ^
	from /usr/share/rvm/rubies/ruby-2.4.3/bin/irb:11:in `<main>'
```

[the remark this answers is not in the document]
Comment (crantok) [reply db/migrate/20181009162402_add_organisation_relationships_table.rb]: Just tested on ruby 2.5.1 and mostly the same behaviour. Making *syntax* validity dependent on *runtime* state just seems incredibly dumb to me.

**Pull request closed without merging**